python/generated/docstrings.h

```diff
@@ -875,19 +880,43 @@ static const char *__doc_bbp_sonata_SimulationConfig_Report_variableName = R"doc
 
 static const char *__doc_bbp_sonata_SimulationConfig_Run = R"doc(Parameters defining global simulation settings for spike reports)doc";
 
+static const char *__doc_bbp_sonata_SimulationConfig_Run_IntegrationMethod = R"doc()doc";
+
+static const char *__doc_bbp_sonata_SimulationConfig_Run_IntegrationMethod_euler = R"doc()doc";
+
+static const char *__doc_bbp_sonata_SimulationConfig_Run_IntegrationMethod_invalid = R"doc()doc";
+
+static const char *__doc_bbp_sonata_SimulationConfig_Run_IntegrationMethod_nicholson = R"doc()doc";
+
+static const char *__doc_bbp_sonata_SimulationConfig_Run_IntegrationMethod_nicholson_ion = R"doc()doc";
+
+static const char *__doc_bbp_sonata_SimulationConfig_Run_SpikeLocation = R"doc()doc";
+
+static const char *__doc_bbp_sonata_SimulationConfig_Run_SpikeLocation_AIS = R"doc()doc";
+
+static const char *__doc_bbp_sonata_SimulationConfig_Run_SpikeLocation_invalid = R"doc()doc";
+
+static const char *__doc_bbp_sonata_SimulationConfig_Run_SpikeLocation_soma = R"doc()doc";
+
 static const char *__doc_bbp_sonata_SimulationConfig_Run_dt = R"doc(Integration step duration in milliseconds)doc";
 
-static const char *__doc_bbp_sonata_SimulationConfig_Run_randomSeed = R"doc(Random seed)doc";
+static const char *__doc_bbp_sonata_SimulationConfig_Run_forwardSkip =
+R"doc(Run without Stimulus or Reports for given duration prior to t=0 using
+a timestep computed as dt=0.1*forward_skip. Default = None)doc";
```

Comment:
Not sure I would bother too much about wrapping lines, it's also not great to lose indentation. Maybe if you wish
```suggestion
static const char *__doc_bbp_sonata_SimulationConfig_Run_forwardSkip =
    R"doc(Run without Stimulus or Reports for given duration prior to t=0 using a timestep computed as dt=0.1*forward_skip. Default = None)doc";
```

Reply:
These are machine generated, so don't need to be reviewed / changed.